version/version_test.go

```diff
@@ -25,12 +25,13 @@ build tags:
 
 func TestInfo_String(t *testing.T) {
 	info := version.Info{
-		Name:      "testapp",
-		AppName:   "testappd",
-		Version:   "1.0.0",
-		GitCommit: "1b78457135a4104bc3af97f20654d49e2ea87454",
-		BuildTags: "netgo,ledger",
-		GoVersion: "go version go1.14 linux/amd64",
+		Name:             "testapp",
+		AppName:          "testappd",
+		Version:          "1.0.0",
+		GitCommit:        "1b78457135a4104bc3af97f20654d49e2ea87454",
+		BuildTags:        "netgo,ledger",
+		GoVersion:        "go version go1.14 linux/amd64",
+		CosmosSdkVersion: "0.42.5",
```

Comment:
btw, will this test fail once we tag 0.42.6?

Reply:
since we dont check sdk version below it should work fine.